### ISSUE TITLE
Initial color not set correctly when pickr parent is not visible.

### ISSUE THREAD

**Issue by DEfusion** [Simonwep/pickr#226]
#### What is the current behavior?

If the `container` the pickr is output in is not visible when it is created the position of the movable handles is not shown correctly on opening, this then has the knock-on effect of not setting the color correctly on interaction unless you click on the palette or a swatch first as can be seen in this video: https://share.getcloudapp.com/d5uWDWLB

#### Please provide the steps to reproduce and create a [JSFiddle](https://jsfiddle.net/Simonwep/qx2Lod6r/).

https://jsfiddle.net/5szxbhqk/

#### What is the expected behavior?

- The elements of the pickr should have their controls in the correct position
- Changing the value by using the hue or alpha slider should set the color correctly

#### Your environment:
```
Version (see Pickr.version): 1.7.1
Used bundle (es5 or normal one): es5
Used theme (default is classic): default
Browser-version:  Chrome 84
Operating-system:  OS X
```

**Comment by Simonwep** [comment]
Fixed in 0d445e5774e23d0e8bc0a694109ea477463820ab, I'll publish a patch version in the next days!

**Comment by Simonwep** [comment]
I've published [1.7.2](https://github.com/Simonwep/pickr/releases/tag/1.7.2) (Mainly because of a recent serious issue), I'll close this issue as we can work in the PR and the current fix is deployed. Thanks again for your efforts, this should be worked on! :D
